src/proto/FunctionRpc.proto

```diff
@@ -180,6 +184,16 @@ message WorkerStatusRequest{
 message WorkerStatusResponse {
 }
 
+message FunctionEnvironmentLoadRequest {
```

Comment:
I think you might have missed renaming the message names!

Reply:
duh..yes. thanks!